compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -60,6 +60,12 @@ int entry(int argc, char **argv)
   arser.add_argument("--all").nargs(0).required(false).default_value(false).help(
     "Enable all optimize options");
 
+  arser.add_argument("--fold_add_v2")
+    .nargs(0)
+    .required(false)
+    .default_value(false)
+    .help("This will fold AddV2 operator");
```

Comment:
(optional) would it be better to express like `This will fold AddV2 operator with both inputs constant` ?

Reply:
I've updated the help message. Thanks.